pinot-core/src/main/java/com/linkedin/pinot/core/query/scheduler/QueryScheduler.java

```diff
@@ -134,16 +134,28 @@ public void stop() {
     byte[] responseData = serializeDataTable(queryRequest, dataTable);
 
     // Log the statistics
+    long numDocsScanned = 0;
+    long numEntriesScannedInFilter = 0;
+    long numEntriesScannedPostFilter = 0;
+    try {
+      numDocsScanned = Long.parseLong(getMetadataValue(dataTableMetadata, DataTable.NUM_DOCS_SCANNED_METADATA_KEY));
+      numEntriesScannedInFilter = Long.parseLong(getMetadataValue(dataTableMetadata, DataTable.NUM_ENTRIES_SCANNED_IN_FILTER_METADATA_KEY));
+      numEntriesScannedPostFilter = Long.parseLong(getMetadataValue(dataTableMetadata, DataTable.NUM_ENTRIES_SCANNED_POST_FILTER_METADATA_KEY));
+      serverMetrics.addMeteredTableValue(queryRequest.getTableNameWithType(), ServerMeter.NUM_DOCS_SCANNED, numDocsScanned);
```

Comment:
(nit) Let's extract "queryRequest.getTableNameWithType()" into local variable because it's used several times.